### DISCUSSION TITLE
Flow: Insert Data of Data Type Json

### DISCUSSION THREAD

**Discussion by CHSchuepfer** [directus/directus#15749] [Archive]
Hey everyone,

im currently struggeling with a problem thats prob. super easy but i cant get it to work.

I have a table with a colum of type "Json" (Postgres v13 - Table was created & configured within directus as Dropdown).
I want a my flow to create a new Dataset in this table.

But everything i tried ends with an 'null' value in the Database.
Here is how my Create Data Operation is configured in terms of payload:
![2022-09-26 10_18_42-Window](https://user-images.githubusercontent.com/4923865/192227749-2eed2d50-47bc-4a64-a321-c383ad79faaf.png)

Here is the resultset directly from the Database:

![2022-09-26 10_20_47-Window](https://user-images.githubusercontent.com/4923865/192228131-ab6e7bce-8f33-4acb-adff-657a4e4e99ac.png)

Does anyone know why this happens, or to be more precise, how i can insert data for a Json Data Type.

Thanks and so long

**Comment by azrikahar** [comment]
Could it be your table column is called `StageSpecifc` (notice the missing letter `i`), but your creation payload is using `StageSpecific`?

**Reply by CHSchuepfer** [reply]
# -> 😭 <-


Thanks for the help dont know how i missed it but apparently i did.
Well better fix it and go back to bed 😆 .


Thanks and so long
